integration-tests/tests/workspace-dashboard.js

```diff
@@ -105,21 +171,32 @@ const testAzureWorkspace = withUserToken(async ({ page, token, testUrl }) => {
   // Must load page before setting mock responses.
   await page.goto(testUrl)
   await findText(page, 'View Workspaces')
+  await overrideConfig(page, { isAnalysisTabVisible: true })
```

Comment:
IA has enabled the Analyses Tab in Azure workspaces, so adding coverage of that.